cli-paho-java/src/main/java/com/redhat/mqe/amc/Receiver.java

```diff
@@ -64,6 +64,9 @@ public void startClient() throws MqttException {
             throw e;
         } catch (InterruptedException e) {
             e.printStackTrace();
+        } catch (Exception ex) {
+            ex.printStackTrace();
+            System.exit(1);
```

Comment:
why is this necessary? can't you let there be uncaught exception? it will stop jvm too, print stacktrace, and exit code will be 1

Reply:
Without it, when receiver is connected and you shut down broker, receiver throws an error because of lost connection and it doesn't shutdown properly.

Reply:
So, the receiver stays running, does not shut down at all, in that case?

Reply:
Yes, exactly.

Reply:
Ok, then ship it! (merge it, I mean)

Reply:
Aye! Thanks.